testutils/clustermanager/e2e-tests/common/common_test.go

```diff
@@ -68,7 +67,7 @@ func TestGetRepoName(t *testing.T) {
 			"a/b/c/", nil, "c", nil,
 		}, {
 			// Git error
-			"", fmt.Errorf("git error"), "", fmt.Errorf("failed git rev-parse --show-toplevel: 'git error'"),
+			"", fmt.Errorf("git error"), "", fmt.Errorf("failed git rev-parse --show-toplevel: git error"),
```

Comment:
kind of surprised these didn't fail a linter for using Errorf instead of errors.New 🤔